roles/identity-management/manage-rh-sso-identities/templates/manage-users.json.j2

```diff
@@ -0,0 +1,9 @@
+{
+    "username": "{{ user_data.user_name | mandatory }}",
+    "firstName": "{{ user_data.first_name | default('') }}",
+    "lastName": "{{ user_data.last_name | default('') }}",
+    "email": "{{ user_data.email | default('') }}",
+    "enabled": {{ user_data.enabled | default(true) }},
+    "emailVerified": {{ user_data.email_verified | default(false) }},
+    "groups": {{ user_data.user_groups | default ('') }}
```

Comment:
Same as above. Not sure we want to include the groups details in this PR since we don't have the group setup configured yet.

Reply:
Yeah, I agree. I think it makes sense to leave it out from this PR.